### PR TITLE
add step to delete the old minikube ISO if present

### PR DESCRIPTION
**Pull request by mdelagrange**
otherwise you may run into issues like kubernetes/minikube#1492

*Issue #, if available:*

*Description of changes:*

Documenting an error that pops up when you upgrade minkube after starting it with a previous version: https://github.com/kubernetes/minikube/issues/1492 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: getting-started/readme.adoc [hunk 1]
```diff
@@ -44,6 +44,10 @@ Verify the version:
     $ minikube version
     minikube version: v0.23.0
 +
+If `minikube` has already been installed earlier, you will also want to delete the previous minikube ISO:
```
Comment (arun-gupta) [inline getting-started/readme.adoc]: The flow of the instructions at https://github.com/mdelagrange/aws-workshop-for-kubernetes/blob/b9bf5cf9ef53513ad0dec22fc7b518171e7bc1fd/getting-started/readme.adoc#setup-on-macos does not seem logical.

. Install minikube
. If already installed, then update
. If error then give a `sudo cp` command
. If already installed, then delete the ISO

Can you please look through the instructions and keep them simple?

Comment (mdelagrange) [reply getting-started/readme.adoc]: Sure, I'll give it another look.

**Comment by arun-gupta** [comment]
Can you resolve the conflicts?

**Comment by mdelagrange** [comment]
OK it should be up to date now.

**Pull request merged**